packages/astro-angular/src/client.ts

```diff
@@ -40,6 +43,33 @@ export default (element: HTMLElement) => {
           }
         }
 
+        if (mirror?.outputs.length && props?.['data-analog-id']) {
+          const destroyRef = appRef.injector.get(DestroyRef);
+          element.setAttribute(
+            'data-analog-id',
+            props['data-analog-id'] as string
+          );
+
+          mirror.outputs.forEach(({ templateName, propName }) => {
+            const outputName = templateName || propName;
+            const component = componentRef.instance as Record<
+              string,
+              Observable<unknown>
+            >;
+            component[outputName]
```

Comment:
Just a note but we'll need to come up with something to support signal-based inputs in the future.